schemas/context/media-timed-asset-reference.schema.json

```diff
@@ -26,6 +26,11 @@
           "type": "string",
           "description": "The friendly, human-readable name of the timed media asset."
         },
+        "id3:TPUB": {
+          "title": "Record Label",
+          "description": "The name of the record label associated with the music recording or video.",
+          "$ref": "http://id3.org/ns/2.4/TPUB"
```

Comment:
The $ref is intended to point to a schema in this github repo with $id=http://id3.org/ns/2.4/TPUB. You cannot reference an external URL or something does that not exist.

Reply:
Was just talking about this one this morning @cmathis .   Why can't it be used to refer to an external namespace?

Reply:
Fixed this